pyproject.toml

```diff
@@ -33,6 +33,7 @@ dev = [
     "pytest",
     "ruff",
     "structlog==22.3.0",
+    "nomad-lab[infrastructure]>=1.3.4dev",
```

Comment:
Why do we need the infrastructure dependencies?

Reply:
Some of the dependencies from `nomad` are needed when running the cap log fixture, specifically: 
```toml
    "python-logstash==0.4.6",
    "mongoengine>=0.20",
    "pyjwt[crypto]==2.6.0",
    "unidecode==1.3.2",
    "fastapi==0.92.0",
    "zipstream-new==1.1.5",
```
Since all of them were mentioned in the `infrastructure`, I thought of adding the full set to avoid future import errors if a new dependency is added, which the cap log fixture might require. However, we can also list only the dependencies that are currently required.